pageserver/src/tenant/mgr.rs

```diff
@@ -665,7 +716,7 @@ pub(crate) async fn upsert_location(
     deletion_queue_client: DeletionQueueClient,
     ctx: &RequestContext,
 ) -> Result<(), anyhow::Error> {
-    info!("configuring tenant location {tenant_id} to state {new_location_config:?}");
+    info!(%tenant_id, "configuring tenant location {tenant_id} to state {new_location_config:?}");
```

Comment:
The `instrument` above already inclues tenant_id. But, it's broken. It needs to be

`field(%tenant_id, %new_location_config)`.

Otherwise, the field will be an "empty field" (i.e., a placeholder) to be filled later in the function body.
Learn more at https://docs.rs/tracing/latest/tracing/attr.instrument.html#adding-fields